opbeat/instrumentation/packages/dbapi2.py

```diff
@@ -56,7 +56,7 @@ def _scan_for_table_with_tokens(tokens, keyword):
             else:
                 return lexeme
 
-        if lexeme == keyword:
+        if isinstance(lexeme, str) and lexeme.upper() == keyword:
```

Comment:
May I suggest

    isinstance(lexeme, six.integer_types)

for enhanced Python3 / unicode compatibility? :)